fpgaconvnet/models/partition/auxiliary.py

```diff
@@ -3,98 +3,126 @@
 
 import fpgaconvnet.tools.graphs as graphs
 import fpgaconvnet.tools.matrix as matrix
-import fpgaconvnet.tools.onnx_helper as onnx_helper
 
 from fpgaconvnet.models.layers import SqueezeLayer
-
 from fpgaconvnet.tools.layer_enum import LAYER_TYPE
 
 def add_squeeze(self):
     # find mismatching streams
+    # print("Add Squeeze called")
     streams_matrix = matrix.get_streams_matrix(self.graph)
     edge_list = matrix.get_edge_list_matrix(self.graph)
-    err = np.sum(streams_matrix,axis=1)
+    err = np.sum(streams_matrix,axis=1)[1:-1]
```

Comment:
This is the main change. This `err` variable is the difference between each stream. The input and output nodes are addressed later on, so this only focuses on the in between streams.

Reply:
Alright, I see. This seems okay to me. As the rest of the code changes below,
I like the introduction of `remove_node_by_type` method. I guess this can be even more general?
Like adding that to `tools/graphs.py` or to some other utils-like file.

Reply:
Yeah I'm happy with this!